### PR TITLE
London Class 7 - Laura M - Git and Github - Week 1

### PR DESCRIPTION
**Pull request by Laura-TM**
My changes to the homework file.

# Your Details

- Your Name: Laura M
- Your City: London
- Your Slack Name: Laura M

# Homework Details

- Module: Git and Github
- Week: 1

### CODE REVIEW

**Comment by Sarah-Nahas** [comment]
well done 👍

**Comment by Sarah-Nahas** [comment]
Hi Laura,
well done this is brilliant.

File: homework.md [hunk 1]
```diff
@@ -4,110 +4,110 @@
...
 
 ## Question 8
 
-What three problem does Git & Github solve?
+What three problems does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1) It makes it easy to contribute to your open source projects
+2) Track changes in your code across versions
+3) Create a database for documentation
```
Comment (Domenico-cyf) [inline homework.md:53]: Hi Laura, Those are  great Answers! Really spot on

**Comment by jmodiale** [comment]
Good job Laura, I noticed you didn't fill in the details on how to correctly title the pull requests but other than that, I loved your answers.

Great job!!

**Pull request closed without merging**